tests/foreman/cli/test_repository.py

```diff
@@ -86,6 +89,7 @@
 from robottelo.helpers import get_data_file
 from robottelo.host_info import get_host_os_version
 from robottelo.test import CLITestCase
+import pathlib
```

Comment:
usually `import` goes above `from *` statements

Reply:
Make sense to me <code>from pathlib import Path<code> should work ?

Reply:
whatever you like, you can as you said or just put `import pathlib` above, like:
```
import pathlib

from fauxfactory import gen_alphanumeric, gen_string
from robottelo import ssh
```

Reply:
Updated code base as per review comments.